src/main/java/stelle/Parser.java

```diff
@@ -0,0 +1,289 @@
+package stelle;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import stelle.exception.DeadlineNoDescriptionException;
+import stelle.exception.DeletionNotSpecifiedException;
+import stelle.exception.EventNoDescriptionException;
+import stelle.exception.MarkNotSpecifiedException;
+import stelle.exception.NoSuchTaskException;
+import stelle.exception.StelleException;
+import stelle.exception.TaskException;
+import stelle.exception.ToDoNoDescriptionException;
+import stelle.exception.WrongCommandException;
+import stelle.task.Deadline;
+import stelle.task.Event;
+import stelle.task.Task;
+import stelle.task.ToDo;
+
+/**
+ * This class encapsulates chatbot parsing and logic.
+ * @author Lee Ze Hao (A0276123J)
+ */
+
```

Comment:
Well done In the organisation of your code, but there are some slight violations on the javadoc conventions. e.g. there should be a line separated before @param and other tags, there should not be a blank line between the comment block and method.